### PR TITLE
Closes #6249 Removed placeholder and added default json

### LINKED ISSUE #6249: [OTHER] Playground Page - Have profile json's placeholder text as default input value for better UX
### What would you like to share?

The placeholder text in profile json disappears as soon as user starts typing. The user has to clear the input text to see the placeholder text again.

I think if we have the default input value as the placeholder text, that would be very simple for the user. The user doesn't have to clear the input text repeatedly to see the format of the profile json.

We can go with that, or either have the profile json sample below the input value so that the user can always refer to it.

https://user-images.githubusercontent.com/27160715/233563200-1f4f514a-b54b-460f-ad10-d43e9eff5e54.mov

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Zhreyu**
## Fixes Issue

Fixes #6249

## Check List (Check all the applicable boxes) 

- [ ✔  ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ✔  ] This PR does not contain plagiarized content.
- [  ✔ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers
Please correct me if I have done any mistake as I'm very new to this.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6294"></a>

### CODE REVIEW

File: pages/playground.js [hunk 1]
```diff
@@ -7,15 +7,28 @@ import Button from "@components/Button";
...
-  const [profileJson, setProfileJson] = useState("");
+  const [profileJson, setProfileJson] = useState(defaultJson);
   const [validateComplete, setValidateComplete] = useState(false);
   const [formatComplete, setFormatComplete] = useState(false);
   const [errorMessage, setErrMsg] = useState("");
   const [successMessage, setSuccessMsg] = useState("");
   const [gitUsername, setGitUsername] = useState("");
   const [previewModalState, setPreviewModalState] = useState(false);
   const [previewModalData, setPreviewModalData] = useState();
+
```
Comment (eddiejaoude) [inline pages/playground.js:31]: Is this extra blank line needed?

File: pages/playground.js [hunk 2]
```diff
@@ -7,15 +7,28 @@ import Button from "@components/Button";
 import PreviewModal from "@components/modals/ProfilePreview";
 import Input from "@components/form/input";
 
+  const defaultJson = `{
+  "name": "Your Name",
```
Comment (eddiejaoude) [inline pages/playground.js:11]: Formatting looks a bit off, please use the VScode Prettier plugin

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good 👍 , but the formatting doesn't look correct

**Comment by Zhreyu** [comment]
Used prettier to format the code!

File: pages/playground.js (hunk 2 above)
Comment (Zhreyu) [reply pages/playground.js:10]: Sure, Thanks for reviewing it

File: pages/playground.js [hunk 3]
```diff
@@ -1,22 +1,32 @@
 import { useState } from "react";
-
 import PageHead from "@components/PageHead";
 import Page from "@components/Page";
 import Alert from "@components/Alert";
 import Button from "@components/Button";
 import PreviewModal from "@components/modals/ProfilePreview";
 import Input from "@components/form/input";
 
+const defaultJson = `{
```
Comment (eddiejaoude) [inline pages/playground.js:9]: This should go in the function below, the scope doesn't need to be wider

**Comment by eddiejaoude** [comment]
Please disable your GitHub Actions on your fork, you should not enable these, it will cause conflicts and also it is sending me notifications

**Comment by eddiejaoude** [comment]
I will fix the failing test via a temporary branch

**Pull request merged**